site/src/_pages/docs/0_index.mdx

```diff
@@ -1,12 +1,26 @@
 ## Overview
 
-These pages provide a practical introduction to implementing the block protocol, whether you are:
+The Block Protocol is a global standard for building interactive blocks connected to structured data.
 
-- interested in [developing blocks](https://blockprotocol.org/docs/developing-blocks), or
-- interested in [embedding blocks in your application](https://blockprotocol.org/docs/embedding-blocks)
+The following pages provide a practical introduction for developers who want to:
+
+- [develop blocks](https://blockprotocol.org/docs/developing-blocks) and publish them to the block registry
+- [embed blocks](https://blockprotocol.org/docs/embedding-blocks) in their own application
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
+### What's a block?
+
+Blocks are the basic units we use to build pages on the modern web. They are rectangular, full-width sections stacked on top of one another – also known as "block-level" elements.
+
+Blocks can be created using plain HTML, or by using a block editor interface like Notion or Wordpress' Gutenberg. With block editors you can usually rearrange the order of blocks on a page using drag and drop.
+
+Each block has a type that determines how it displays data. You're probably familiar with the most common block types such as _paragraph_, _header_, _image_, _list_, and _table_. Some block types have more advanced interactive features such as filtering, search, and sorting.
+
+There are an inifinite number of possible block types. One goal of the Block Protocol is to encourage developers to build and share brand new block types we've never seen before.
+
+From a developer perspective, blocks are similar to [web components](https://en.wikipedia.org/wiki/Web_Components). They are reusable sets of HTML elements that can accept any data that matches the block's schema. For example, an image block can only accept an image URL or file, but not a text string.
```

Comment:
I'm assuming this is talking about generic blocks, rather than BP blocks - we could add the sentence I suggest above after this ("A block which complies..."). 

This could be rephrased but I'm trying to get across that:
- web components are an implementation method rather than an alternative to blocks
- "schema" is something we use rather than being universally in use for different versions of "blocks"
- pedantry: tweaked the example to make it starker, as technically an image URL is a text string
 
```suggestion
From a developer perspective, blocks offer encapsulation of specific functionality ([web components](https://en.wikipedia.org/wiki/Web_Components) are one possible way of implementing a block). They are reusable elements that expect certain data to be passed to them (what we refer to as a block's "schema"). For example, an image block might only accept an image URL in its `src` property, not a number.
```

Reply:
I'd also move the "goals of the BP" to the bottom as I think it'd flow better with a new para on BP blocks. So I'm suggesting the final paras would be something like this. Maybe with the paras swapped?

> There are an infinite number of possible block types. One goal of the Block Protocol is to encourage developers to build and share brand new types of blocks that haven't been seen before, and for applications to offer users far more blocks than any one team could imagine or build.
> 
> A block which complies with the Block Protocol has the added advantage of specifying the data it receives - and modifying it - in a standardized way. This makes it easier to re-use Block Protocol blocks across applications - and if they are published to the Block Hub, easily discoverable by users.